frontend/src/features/properties/map/activity/detail/ActivityPropertyModal.test.tsx

```diff
@@ -58,8 +58,17 @@ describe('ActivityPropertyModal tests', () => {
   });
 
   it('Renders as expected', async () => {
-    const { asFragment } = setup();
-    expect(asFragment()).toMatchSnapshot();
+    setup();
+    expect(document.body).toMatchSnapshot();
+  });
```

Comment:
For modals we need to snapshot the whole document body or we just basically get an empty snapshot